### ISSUE TITLE
Server: Reg-Req-7 F-7: expected result processing invalid

### ISSUE THREAD

**Issue by jenkay** [fido-alliance/conformance-tool-issues#7]
The test case F-7 (If AAIDs of all disallowed authenticators is not added to m.aaid) under Reg-Req-7 (Registration Request Generation Rules for FIDO UAF Server)

```
AssertionError: m.policy.disallowed does matchCriteria for each of the registered authenticator. 
Expected [
{
"aaid": [
"FFFF#FC01",
"FFFF#FC02",
"FFFF#FC03"
],
"keyIDs": [
"sL6rc7sV1MqGEqGQTTwklg56tPI-u7UA1dkNtn_HXY0",
"0wfOp-FZljRN67fh-YQpchxhFDR511W7ngrQ3AoLaRs",
"usVzdt5W35FMMa9khmYRW8zJdyoBgnhCAWVplkgxuXg"
]
}
]. Got [
{
"aaid": [
"FFFF#FC02"
],
"keyIDs": [
"0wfOp-FZljRN67fh-YQpchxhFDR511W7ngrQ3AoLaRs"
]
},
{
"aaid": [
"FFFF#FC01"
],
"keyIDs": [
"sL6rc7sV1MqGEqGQTTwklg56tPI-u7UA1dkNtn_HXY0"
]
},
{
"aaid": [
"FFFF#FC03"
],
"keyIDs": [
"usVzdt5W35FMMa9khmYRW8zJdyoBgnhCAWVplkgxuXg"
]
}
]
==
```
The expected result disallows three authenticators ("FFFF#FC01", "FFFF#FC02","FFFF#FC03) combined with ANY of the KeyIDs. But only the combination of AAID+KeyID is considered unique.

The result that was provided by the server correctly disallows pairs of AAIDs and related KeyIDs.

The test tool should be corrected.

See section 3.1.11 in UAF Protocol and 3.1.5 ("The (AAID, KeyID ) tuple must uniquely identify an authenticator's registration for a relying party.")

**Comment by herrjemand** [comment]
Awaits TWG/CWG discussion

**Comment by herrjemand** [comment]
Resolved in v0.9.12
